CSharp/DevVmPowershell/Helpers/Implementations/DisclaimerAcceptanceHelper.cs

```diff
@@ -252,26 +306,37 @@ private int GetDisclaimerObjectTypeId(int workspaceId)
 			return objectTypeId;
 		}
 
-		private int GetWorkspaceId(string workspaceName)
+		private async Task<int> GetWorkspaceIdAsync(string workspaceName)
```

Comment:
Should we move this API call to the Workspace helper?

Reply:
Done